templates/openstackclient/bin/init.sh

```diff
@@ -44,6 +44,9 @@ if [ -d /mnt/ssh-config ]; then
   chown -R cloud-admin: /home/cloud-admin/.ssh
 fi
 
+# make the ctlplane ansible inventory executable
+chmod 755 /home/cloud-admin/ctlplane-ansible-inventory
```

Comment:
Is this necessary? The ConfigMapVolumeSource default mode is 0755.

Reply:
Good call. This was actually what broke it so removing it fixes everything.